Examples/UIExplorer/SliderExample.js

```diff
@@ -167,4 +167,30 @@ exports.examples = [
       );
     }
   },
+  {
+    title: 'Custom thumb image',
```

Comment:
There is already an iOS example for this prop we can just reuse it by removing the `platform: ios`.